taichi/transforms/scalarize.cpp

```diff
@@ -378,10 +378,12 @@ class ScalarizePointers : public BasicStmtVisitor {
 
 namespace irpass {
 
-void scalarize(IRNode *root) {
+void scalarize(IRNode *root, const CompileConfig &config) {
```

Comment:
nit: another way to obtain `CompileConfig`: `root->kernel->program->this_thread_config()`, so you dont need to change the function signature.

Reply:
Updated. Thanks!